tests/convergence/ex1c.cpp

```diff
@@ -0,0 +1,414 @@
+//                                MFEM Example 1 - complex
+//
+// Compile with: make ex1c
+//
+//  ./ex1c -m ../../data/inline-hex.mesh -o 1 -ref 4 -sol 1 -iprob 1 -no-vis
```

Comment:
The `-ref 4` option seems a bit large for this serial example.

Reply:
I did not review the convergence tests carefully (other than to make sure they worked correctly), since it is stated that they will not be merged (just provided for the reviewers). @psocratis is this still the plan?

Reply:
Well, I don't really know. This was the original intention. Eventually, convergence tests will be added but I don't think it would be part of this PR

Reply:
changed to 2